examples/treegrid/treegrid-1.html

```diff
@@ -83,18 +77,8 @@ <h1>Treegrid Email Inbox Example</h1>
     The following example demonstrates how the
```

Comment:
On line 36 of [Example-Template.html](https://github.com/w3c/aria-practices/blob/master/examples/coding-template/Example-Template.html) there is a set of four links in a nav region that are missing from this page.